ios/FluentUI/Shimmer/ShimmerView.swift

```diff
@@ -5,6 +5,35 @@
 
 import UIKit
 
+/// Shimmer style can be either concealing or revealing
+/// The style affects the default shimmer alpha value and the default shimmer tint color
+@objc(MSFShimmerViewStyle)
+public enum ShimmerStyle: Int, CaseIterable {
+    /// Concealing shimmer: the gradient conceals parts of the subviews as it moves leaving most parts of the subviews unblocked.
+    case concealing
+
+    /// Revealing shimmer: the gradient reveals parts of the subviews as it moves leaving most parts of the subview blocked.
+    case revealing
+
+    var defaultAlphaValue: CGFloat {
+        switch self {
+        case .concealing:
+			return 0.4
+        case .revealing:
+			return 0.6
+        }
+    }
+	
+	var defaultTintColor: UIColor {
+		switch self {
+		case .concealing:
+			return .white
```

Comment:
This needs to be a constant on the colors with defined values for both light and dark modes.

![Screen Shot 2020-09-24 at 11 07 08 PM](https://user-images.githubusercontent.com/68076145/94232395-ba3aa880-feba-11ea-8db4-617008843d82.png)

Reply:
Added dark variants to the demo images and dark mode colors.